wordpress-mobile/WordPress-iOS
# Share Extension: Improve TextBundle/Pack import

Fixes #11223

*Also may fix these issues:*:
Refs #11350
Fixes #10736 (maybe—the caveat is that it ignores max image size entirely atm)
Fixes #9555

This improves our support for TextBundle/Pack support in two ways:
1. Images in the bundle are now added to the post
2. Markdown support is added (via [Down](http://github.com/iwasrobbed/Down), which wraps CommonMark). This also has the side effect of adding Markdown support for import in general.

### To test:
- First, we should ensure that all existing sharing functionality is unharmed…
- To test text bundle support, launch bear:
  - In the list of notes, swipe left on the one you'd like to share
  - Tap more
  - Tap Export note…
  - Tap Text Bundle
  - Tap WordPress
- Ensure that the text is formatted properly, that any included images are included, and that generally things look mostly correct

Update release notes:
